client/src/components/FeedbackForm/FeedbackForm.js

```diff
@@ -0,0 +1,103 @@
+import React, { useMemo } from 'react';
+import { useDispatch, useSelector } from 'react-redux';
+import { Formik, Form } from 'formik';
+import { Alert, Button, FormGroup, Spinner } from 'reactstrap';
+import { FormField } from '_common';
+import * as Yup from 'yup';
+import './FeedbackForm.module.scss';
+
+const formSchema = Yup.object().shape({
+  subject: Yup.string().required('Required'),
+  name: Yup.string().required('Required'),
+  email: Yup.string()
+    .email('Invalid email')
+    .required('Required'),
+  problem_description: Yup.string().required('Required')
+});
+
+const FeedbackForm = () => {
+  const dispatch = useDispatch();
+  const authenticatedUser = useSelector(state => state.authenticatedUser.user);
+  const creating = useSelector(state => state.ticketCreate.creating);
+  const creatingError = useSelector(state => state.ticketCreate.creatingError);
+  const creatingErrorMessage = useSelector(
+    state => state.ticketCreate.creatingErrorMessage
+  );
+
+  if (authenticatedUser == null) {
+    return <Spinner />;
+  }
+
+  const defaultValues = useMemo(
+    () => ({
+      subject: 'Feedback for the Frontera Portal',
+      problem_description: '',
+      name: authenticatedUser
+        ? `${authenticatedUser.first_name} ${authenticatedUser.last_name}`
+        : '',
+      email: authenticatedUser ? authenticatedUser.email : ''
+    }),
+    [authenticatedUser]
+  );
+
+  return (
+    <Formik
+      enableReinitialize
+      initialValues={defaultValues}
+      validationSchema={formSchema}
+      onSubmit={(values, { resetForm }) => {
+        const formData = new FormData();
+        Object.keys(values).forEach(key => formData.append(key, values[key]));
+        dispatch({
+          type: 'TICKET_CREATE',
+          payload: {
+            formData,
+            resetSubmittedForm: resetForm,
+            refreshTickets: true
+          }
+        });
+      }}
+    >
+      {({ isSubmitting, dirty, isValid, submitCount }) => {
+        return (
+          <Form styleName="feedback-form">
```

Comment:
__Consistency__: (JSX) Use standard top-level class name for CSS modules (`.container`, discrepancies will be fixed).

```suggestion
          <Form styleName="container">
```